<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamsWidget</class>
 <widget class="QDialog" name="ParamsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonsBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="mode">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>13</y>
     <width>99</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Mode</string>
   </property>
   <widget class="QRadioButton" name="xorRB">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>31</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>XOR</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="replaceRB">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>58</y>
      <width>73</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fieldSize">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>13</y>
     <width>199</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Field Size</string>
   </property>
   <widget class="QLabel" name="widthL">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>63</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="heigthL">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>32</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="heightSP">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="widthSB">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <zorder>heigthL</zorder>
   <zorder>widthL</zorder>
   <zorder>heightSP</zorder>
   <zorder>widthSB</zorder>
  </widget>
  <widget class="QGroupBox" name="cellProp">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Cell properties</string>
   </property>
   <widget class="QLabel" name="cellL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cell</string>
    </property>
   </widget>
   <widget class="QSlider" name="cellVS">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>55</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>25</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="cellSB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gameRul">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>114</y>
     <width>561</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>GamesRules</string>
   </property>
   <widget class="QLabel" name="fstL">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>31</y>
      <width>74</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FST_IMAPCT</string>
    </property>
   </widget>
   <widget class="QLabel" name="bEndL">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BIRTH_END</string>
    </property>
   </widget>
   <widget class="QLabel" name="lBeginL">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>31</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LIVE_BEGIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="scdL">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>31</y>
      <width>78</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SCD_IMPACT</string>
    </property>
   </widget>
   <widget class="QLabel" name="lEndL">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>31</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LIVE_END</string>
    </property>
   </widget>
   <widget class="QLabel" name="bBeginL">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>85</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BIRTH_BEGIN</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fstB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>66</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scdB">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="lBeginB">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>66</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="lEndB">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>66</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="bBeingB">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="bEndB">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>66</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonsBox</sender>
   <signal>accepted()</signal>
   <receiver>ParamsWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonsBox</sender>
   <signal>rejected()</signal>
   <receiver>ParamsWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
